setup.py

```diff
@@ -339,8 +333,8 @@ def test_otio():
     ),
 
     ext_modules=[
-        CMakeExtension('_opentimelineio'),
-        CMakeExtension('_opentime'),
+        Extension('opentimelineio._otio', sources=[]),
```

Comment:
The full and correct module name is required here because setuptools needs to resolve the name to find the produced file and copy it into the source tree. (Because yes, editable install means that the install should point to the source tree, which means the `.so`s need to also be there alongside the python files).